gcs/conn_test.go

```diff
@@ -1,77 +0,0 @@
-// Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Why are you deleting this unit test?

Reply:
Accidentally made some changes in this file. will again push original one